### DISCUSSION TITLE
Is there a beginners tutorial for this?

### DISCUSSION THREAD

**Discussion by rdong8** [nautechsystems/nautilus_trader#382] [Q&A] [answered]
Hi, is there a beginners tutorial for Nautilus Trader? I looked through the examples but it was a bit overwhelming to figure out how everything pieces together. Does this project have a YouTube channel or something similar?

**Comment by limx0** [comment, marked as answer]
Hey @rdgh8 sorry for the slow response. There aren't really any tutorials (or videos) outside of the examples folder. The examples are fairly self contained and could be copy-pasted into a jupyter notebook if you wanted to play around with them. 

Do you have any specific questions or things you're unsure about? Maybe we could answer them.

**Reply by chungichien0923** [reply]
Is there a beginner's tutorial for Nautilus Trader now?
I'm copying and pasting part of the codes from examples(.py and .ipynb), trying to add my own data(5mins ohlc data) to practice backtesting, but it's hard to understand how to correctly use it...
For example, I find code writing like that "... catalog.write_data(ticks) ..." in external_data_backtest.ipynb. but when I try to use it(though I use bar data instead, not tick data), An error "AttributeError: 'ParquetDataCatalog' object has no attribute 'write_data'" comes out...

Should I read anything first, if I want to use my own bar data to backtest, or even more, to use interactive broker's tick data to paper-trade(on bar data)?

I apologize if I asked anything dumb🥲, I'm kinda newbie.
